lib/facter/resolvers/aix/disks.rb

```diff
@@ -47,7 +47,7 @@ def find_size(name)
           end
 
           def compute_size(size_hash)
-            physical_partitions = size_hash['TOTAL PPs'].to_i + size_hash['FREE PPs'].to_i
+            physical_partitions = size_hash['TOTAL PPs'].to_i
```

Comment:
Is this covered by an existing test? If not, can you add one?

Reply:
I unfortunately do not have any experience with Rspec, but looking at [aix/partitions_spec.rb](https://github.com/puppetlabs/facter/blob/main/spec/facter/facts/aix/partitions_spec.rb) and [aix/mountpoints_spec.rb](https://github.com/puppetlabs/facter/blob/main/spec/facter/facts/aix/mountpoints_spec.rb), it seems to me that the expected size values are checked against MiB/GiB. So as long as the test environment did not use additional logical volume copies (mirrors), these tests would have succeed in the past. As far as I can tell the tests will also work with the proposed code changes in this pull request for mirrored logical volumes.

Reply:
This makes sense. I have no idea why were adding free physical partitions to the total